pkg/registry/populator.go

```diff
@@ -46,30 +50,41 @@ func (i *DirectoryPopulator) Populate(mode Mode) error {
 		return fmt.Errorf("unable to read directory %s: %s", metadata, err)
 	}
 
-	// Look for the metadata and manifests sub-directories to find the annotations.yaml file that will inform how the
-	// manifests of the bundle should be loaded into the database.
+	// Look for the metadata and manifests sub-directories to find the annotations.yaml
+	// file that will inform how the manifests of the bundle should be loaded into the database.
+	// If dependencies.yaml which contains operator dependencies in metadata directory
+	// exists, parse and load it into the DB
 	annotationsFile := &AnnotationsFile{}
+	dependenciesFile := &DependenciesFile{}
 	for _, f := range files {
 		err = decodeFile(filepath.Join(metadata, f.Name()), annotationsFile)
 		if err != nil || *annotationsFile == (AnnotationsFile{}) {
+			log.Info("found annotations file searching for csv")
 			continue
 		}
-		log.Info("found annotations file searching for csv")
+		if f.Name() == DependenciesFileName {
```

Comment:
Didn't we discuss identifying the dependency file based on some identifier in the file?